.travis.yml

```diff
@@ -1,15 +1,18 @@
 language: node_js
 node_js:
+  - "0.12"
+  - "0.11"
   - "0.10"
   - "0.8"
+  - iojs
 script: "npm install && npm test"
 before_install:
   - git submodule update --init --recursive
-  - npm install -g npm@~1.4.6
+  - ruby -e "if '`npm -v`'<'2.0.0' then \`npm i -g npm\` else '' end"
```

Comment:
> So if I understand it correctly Npm > 2.0 is required?

Well, npm should be atleast 1.4.6. But I have trouble trying to do `<` and `>` operations on it. So, I used `2.0.0` which was easier to do those operations. Except `0.8` and `0.10`, all other versions have npm > 2.0.0